release-notes/Release-Notes-RC1.md

```diff
@@ -0,0 +1,70 @@
+# Release Notes
+
+## RC1 released 11/18/2015
+
+Full details on this release can be read about in this blog post:
+[Announcing .NET Core and ASP.NET 5 RC](http://aka.ms/netcorerc)
+
+Supported platforms for this release are Ubuntu 14.04 LTS, OS X 10.10, and Windows 7+.
+
+### Dependencies
+.NET Core RC1 on Linux and OSX take dependencies on the following libraries:
+* libc, librt: I/O, process management, time, etc.
+* libunwind: call chain determination for exception handling and stack traces
+* gettext: resource strings
+* libicu: globalization
+* liblttng: runtime tracing
+* libuuid: guid generation
+* libcurl: HTTP
+* libssl & libcrypto: cryptography
```

Comment:
Please have versions. OSX comes with a `libssl` but we need a higher version of it.